cowprotocol/cowswap
# Widget UI 7

# Summary

- modal header update
- Resize iFrame on parent based on iFrame content height using post message

https://github.com/cowprotocol/cowswap/assets/31534717/9a20444e-1cef-411b-8d25-297e695df831

## Iframe resize parent/widget



https://github.com/cowprotocol/cowswap/assets/31534717/04e43c13-7057-4c95-af4a-6950a76395a4

To test
- Run the cowswap frontend on localhost:3000
- Open this https://jsfiddle.net/rmbf863n/11/ 
- Should adjust the Iframe height to the Iframe's content height.
